client/src/components/MapGeocoderWrapper.vue

```diff
@@ -29,20 +30,27 @@ import mapboxgl from 'mapbox-gl';
  */
 export default defineComponent({
   name: 'MapGeocoderWrapper',
-  setup() {
+  setup(props, { emit }) {
+    const visible = computed({
+      get: () => props.modelValue,
+      set: (value) => emit('update:modelValue', value),
+    });
     const state: State = inject(stateKey, State.default());
 
     return {
       state,
+      visible,
```

Comment:
Can we call this something like `expanded`? I was initially confused since it doesn't literally describe the visibility of the component.

Reply:
yeah for sure